Dockerfile

```diff
@@ -1,192 +1,43 @@
 # Multistage docker build, requires docker 17.05
 
 # builder stage
-FROM ubuntu:16.04 as builder
+FROM ubuntu:20.04 as builder
 
 RUN set -ex && \
     apt-get update && \
-    apt-get --no-install-recommends --yes install \
+    DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends --yes install \
         ca-certificates \
         cmake \
-        g++ \
-        make \
+        build-essential \
+        libtool \
+        autotools-dev \
+        automake \
         pkg-config \
-        graphviz \
-        doxygen \
-        git \
+        bsdmainutils \
         curl \
-        libtool-bin \
-        autoconf \
-        automake \
-        bzip2 \
-        xsltproc \
-        gperf \
```

Comment:
whoops, `gperf` seems to be required actually (not sure why it works fine in github actions though)

Reply:
Seems to be installed here: https://github.com/monero-project/monero/blob/master/.github/workflows/depends.yml#L35

Don't know if the others are also required.

Reply:
oh true

sounds like no - I just finished a build with the addition of just `gperf` 👍

Reply:
No build-essentials / patch?